reana_client/utils.py

```diff
@@ -358,3 +358,20 @@ def parse_secret_from_path(path):
 def get_api_url():
     """Obtain REANA server API URL."""
     return os.environ.get('REANA_SERVER_URL')
+
+
+def get_reana_yaml_file_path():
+    """REANA specification file location."""
+    matches = [path for path in reana_yaml_valid_file_names
+               if os.path.exists(path)]
+    if len(matches) > 1:
+        click.echo(
+            click.style(
+                'You must have only one specification file. '
```

Comment:
Better to make the message less personal, and better to indicate clearly that we are exiting in this case... Something like:

```console
$ reana-client run -w test
==> ERROR: Found two REANA specification files (reana.yaml, reana.yml). Please use only one. Exiting.
```

We could use the same technique when no spec file is found:

```console
$ reana-client run -w test
==> ERROR: No REANA specification file (reana.yaml) found. Exiting.
```

Currently we print:

```console
$ reana-client run -w test
Usage: reana-client run [OPTIONS]
Try "reana-client run --help" for help.

Error: Invalid value for "-f" / "--file": Path "reana.yaml" does not exist.
```

which is perhaps a bit less friendly to the user... 

Otherwise the code looks good and the functionality works well :+1: 

P.S. The output styling  (==>) used above is just for illustration, but we can indeed start generalising it here and there in CLI commands, as we started to introduce it in `logs` command... see e.g. [yay screenshot](https://camo.githubusercontent.com/90b22c7fc8c60528a78f055c93cd33e05a991dd9/68747470733a2f2f72617763646e2e6769746861636b2e636f6d2f4a677565722f6a677565722e6769746875622e696f2f373736343766333936636237313536666433326533303937306462656166366436646337663938332f7961792f7961792d732e706e67).

Reply:
> Better to make the message less personal, and better to indicate clearly that we are exiting in this case... Something like:
> 
> ```
> $ reana-client run -w test
> ==> ERROR: Found two REANA specification files (reana.yaml, reana.yml). Please use only one. Exiting.
> ```
Agreed.

> 
> We could use the same technique when no spec file is found:
> 
> ```
> $ reana-client run -w test
> ==> ERROR: No REANA specification file (reana.yaml) found. Exiting.
> ```
> 
> Currently we print:
> 
> ```
> $ reana-client run -w test
> Usage: reana-client run [OPTIONS]
> Try "reana-client run --help" for help.
> 
> Error: Invalid value for "-f" / "--file": Path "reana.yaml" does not exist.
> ```
> 
> which is perhaps a bit less friendly to the user...

I thought about this at the beginning, but I think then we'd be losing valuable information for the user explaining how to pass a `reana.yaml` manually, WDYT? cc/ @diegodelemos

Reply:
I guess users will mostly use the default file name anyway, as they see it in the docs and other examples. It does not seem probable that someone would not have any `reana.yaml` but would rather insist on using say `.reana.yaml` and passing the `-f` option all the time.  It is more probable that people would have both `reana.yaml` and some other file, such as `reana-quicktest.yaml`. Hence I think it is OK to alter the message when there is no `reana.yaml`; these people would be mostly novices who would like to have `reana.yaml` anyway.  So the shorter message should be OK.

BTW, speaking of `-f', we should upgrade one day `reana-client` help messages for the `-f` flag anyway, since we are not listing there that `reana.yaml` is the default:

```
 -f, --file PATH                 REANA specifications file describing the
                                  workflow and context which REANA should
                                  execute.
```

This would be better simplified into _"REANA specification file describing the workflow to execute. [default=reana.yaml]"_ in every command.

We can leave some of these improvements and cosmetic formatting styles for later, my important comment was about the output message verbiage; all the other remarks about formatting etc were just for bonus points :wink:

Reply:
OK, makes sense to me. Updated and help messages upgraded.